src/Components/DeleteModal.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react'
+import { AiFillDelete } from "react-icons/ai";
+
+
+class DeleteModal extends Component {
+    constructor(props) {
+        super(props)
+    
+        this.state = {
+            showModal: false
+        }
+        this.setShowModal = this.setShowModal.bind(this)
+        this.onDelete = this.onDelete.bind(this)
+    }
+
+    setShowModal(flag){
+        this.setState({
+            showModal: flag
+        })
+    }
+    
+    onDelete(e){
+        const {id, deleteTask} = this.props
+        this.setShowModal(false)
+        deleteTask(id)
+    }
+
+    render() {
+        return (
+            <>
+                <AiFillDelete
+                    className="ml-2 h-6 w-6 cursor-pointer"
+                    onClick={() => this.setShowModal(true)}
+                >
+                    Open small modal
+                </AiFillDelete>
```

Comment:
what is the purpose of this text? if it's necessary then change this text to `Open delete task modal`